tests/integration/tf/asvdb/test_asvdb_transformers_next_item_prediction.py

```diff
@@ -0,0 +1,64 @@
+import platform
+
+import psutil
+from asvdb import ASVDb, BenchmarkInfo, BenchmarkResult, utils
+from testbook import testbook
+
+from tests.conftest import REPO_ROOT
+
+
+@testbook(
+    REPO_ROOT / "examples/usecases/transformers-next-item-prediction.ipynb",
+    timeout=360,
+    execute=False,
+)
+def test_func(tb):
+    tb.inject(
+        """
+        import os
+        os.environ["INPUT_DATA_DIR"] = "/workspace/data"
```

Comment:
Can we check @jperez999 which folder structure we have in our nightly CI? I think we need something like `/raid/data/bookingcom/raw`